docs/compendium/view.md

```diff
@@ -45,7 +43,7 @@ You can also get only the compendia uploaded by a specific user. A user filter c
 - `job_id` - Comma-separated list of related job ids to filter by.
 - `user` - Public user identifier to filter by.
 - `start` - List from specific search result onwards. 1-indexed. Defaults to 1.
```

Comment:
Can you clarify what "1-indexed" means? I cannot :-). Just remove that sentence please.